### ISSUE TITLE
nvda does not le the updated status of the combo box

### ISSUE THREAD

**Issue by fernando-jose-silva** [nvaccess/nvda#8229]
### Steps to reproduce:
open the page:
https://appleid.apple.com/account#!&page=create
press c to go to the first combo box to choose the country.
Press alt + down arrow to open the combo box.
press down arrow to go to the next country in the combo box.
press enter.
press esc to return to the navigation mode.
press the up arrow to read the name of the selected country in the combo box.

### Expected behavior:
nvda should announce the chosen field in the combo box.

### Actual behavior:
nvda announces the name of the country he had filled in the combined box before choosing the new country.
If you update the virtual buffer with nvda + f5 and read the combo box, nvda will read the status in case the correct and currently selected country.
I saw this case on other pages as well.

### System configuration:
NVDA version:
next 15086
NVDA Installed or portable:
instaled
Other information:
> Example: Running in a VM

Windows version:
windows 7 e windows 10 build 17134.1
Name and version of other software in use when reproducing the issue:
firefox 59.03 64bits

### Other questions:

Does the issue still occur after restarting your PC?
yes
Have you tried any other versions of NVDA?
> Please list them and the result
no

**Comment by Adriani90** [comment]
with NVDA 2019.3 Beta 1 and Firefox 72.0.1 the current selected value is not reported when the combo box is colaped. So to hear what I have selecteed, I have to expand the combo box with alt+down arrow.

**Comment by Adriani90** [comment]
This is not reproducible anymore in Firefox 111 with NVDA last alpha version. I am closing as works for me. Please comment on this if you are still having this issue and we can reopen.
